trimmer_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+
+class TrimmerDecorator < Nameable
+  def initialize(nameable = nameable.new)
+    super()
+    @nameable = nameable
+  end
```

Comment:
- [ ] Here `TrimmerDecorator`should  Inherits from the Decorator. && is no need to initialize again. Kindly remove the initialized part.
 ```ruby
class TrimmerDecorator < Decorator
```